.github/workflows/nightly.yml

```diff
@@ -0,0 +1,38 @@
+name: Nightly
+run-name: ${{ github.actor }} triggered nightly on ${{ github.ref }}
+on:
+    schedule:
+      # * is a special character in YAML so you have to quote this string
+      - cron: '0 0 * * *'
+
+    workflow_dispatch:
+
+jobs:
+
+    # multi-gpu
+    AWS-AVX2-192G-4-A10G-96G:
+        strategy:
```

Comment:
oh, we can likely just drop the "strategy" for now, since it only has one element.